### ISSUE TITLE
Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation"  buttons fails to meet the required ratio of 3:1: A11y_VS Code extension for API Center_Function dropdown_Non Text Contrast.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#134]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG1.4.11; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #A11yMAS; #Non Text Contrast; #A11YWCAG2.1; #FirstParty:Bug212317;#External;#External:VSCode;#BM-VisualStudioCodeClient-Win32-Jan2024;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Generate HTTP file" using the down arrow.
7. Verify whether the Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation" buttons fails to meet the required ratio of 3:1

### Actual Result:
Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation" buttons fails to meet the required ratio of 3:1.

### Expected Result:
Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation" buttons should be greater than or equal to required ratio of 3:1.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

### Attachment:
![image](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/ad814967-a0ce-4d5d-aea2-9b5688c4c295)

**Comment by formulahendry** [comment]
External issue. This UI control is from tree view of VS Code: https://github.com/microsoft/vscode

**Comment by Yash14j** [comment]
@formulahendry As per the C+AI accessibility testing external bug process primary team need to file the bug to VSCode team & share us the external bug ID.

**Comment by formulahendry** [comment]
Opened bug in vs code repo: https://github.com/microsoft/vscode/issues/212317

**Comment by AnKushSingh05** [comment]
Issue is fixed in Version: 1.95.2 (user setup)

![Image](https://github.com/user-attachments/assets/0f3b8a55-76d4-4f7f-8cdd-5855b39a7dc1)
